commcare_connect/opportunity/models.py

```diff
@@ -64,9 +64,12 @@ class Opportunity(BaseModel):
         on_delete=models.CASCADE,
         null=True,
     )
+    # to be removed
     max_visits_per_user = models.IntegerField(null=True)
     daily_max_visits_per_user = models.IntegerField(null=True)
+    start_date = models.DateField(null=True, default=datetime.date.today)
```

Comment:
Will the existing opportunities without this field get auto-set to today, they won't right? That's the reason the null is there.

Reply:
According to django docs the default value specified on the models will be used for older table rows. 
https://docs.djangoproject.com/en/5.0/ref/migration-operations/#addfield

Reply:
If we do go with a default (which I think makes more sense than a nullable value because then we can be sure it always exists, unless this is necessary for the creation workflow), I think it makes more sense to use the opportunity creation date, rather than today if possible.

Reply:
This makes sense, I will remove null=True. 
>  I think it makes more sense to use the opportunity creation date, rather than today if possible.
Right, for old opportunities. For new ones, it would be today